### DISCUSSION TITLE
Is it possible to install and run without using Docker?

### DISCUSSION THREAD

**Discussion by charleslcso** [ghostfolio/ghostfolio#1100] [Q&A]
Hello,

Is it possible to install and run ghost folio without using Docker?

**Comment by dtslvr** [comment]
Hi @charleslcso 

Local use of Ghostfolio without Docker is theoretically possible, but only recommended for developers, if at all. You would need to run the build and manage the database (Postgres) / cache (Redis) instances by yourself.

I would recommend to check out the managed Ghostfolio [cloud offering](https://ghostfol.io/pricing).

**Reply by charleslcso** [reply]
Ok thank you.
